app/models/prompt.rb

```diff
@@ -222,17 +222,6 @@ def fulfilled_claims
     self.request_claims.fulfilled
   end
 
-  # We want to have all the matching methods defined on
-  # TagSet available here, too, without rewriting them,
-  # so we just pass them through method_missing
-  def method_missing(method, *args, &block)
-    super || (tag_set && tag_set.respond_to?(method) ? tag_set.send(method) : super)
-  end
-
-  def respond_to?(method, include_private = false)
-    super || tag_set.respond_to?(method, include_private)
-  end
```

Comment:
I deleted this because it didn't seem to work right with the internals of the Bullet gem.

The reason I think it's safe to delete (and the reason it doesn't play well with Bullet) is because this way of overriding `method_missing` doesn't do anything -- calling `super` as the first part of the `method_missing` definition will just cause it to raise a `NoMethodError` if the `Prompt` doesn't have a method by that name, because that's what the default `method_missing` does. So the rest of the `method_missing` code here is unreachable.

(It's also not listed as covered on codecov.io.)